azkaban-common/src/test/java/azkaban/executor/ExecutorManagerTest.java

```diff
@@ -72,8 +72,9 @@ private ExecutorManager createMultiExecutorManagerInstance(
 
     loader.addExecutor("localhost", 12345);
     loader.addExecutor("localhost", 12346);
-    return new ExecutorManager(props, loader, new AlerterHolder(props),
-        new CommonMetrics(new MetricsManager(new MetricRegistry())));
+    final CommonMetrics commonMetrics = new CommonMetrics(new MetricsManager(new MetricRegistry()));
```

Comment:
Should we iniatialize guice `@before` any test, so that we can use Guice to get ExecutorManager?